lib/sidekiq_unique_jobs/orphans/manager.rb

```diff
@@ -39,7 +39,9 @@ def start(test_task = nil) # rubocop:disable
         self.task = test_task || default_task
 
         with_logging_context do
-          register_reaper_process
+          got_lock = register_reaper_process
```

Comment:
If you have a look at the top of the method, we check if a manager has been registered already so unless I am completely missing something (happens more often than I'd like to admit); this shouldn't be necessary?

Reply:
Ahh, yes, thank you for explaining! I'll just close this one =)